services/publishing/base-repo/security/policy.json

```diff
@@ -0,0 +1,90 @@
+{
+    "policies": [
+        {
+            "group": "*",
```

Comment:
duplicate

Reply:
Actually it isn't, this one has no contexts, the second one has any context.

Reply:
But context "*" will hit everything: 
https://github.com/Soluto/tweek/blob/master/services/gateway/authorization.rego#L18